### ISSUE TITLE
inst_loss: nan

### ISSUE THREAD

**Issue by spatiall** [xuxw98/ESAM#17]
Hello author, when I run this command, CUDA_VISIBLE_DEVICES=0 python tools/train.py configs/ESAM_CA/ESAM_sv_scannet200_CA.py --work-dir work_dirs/ESAM_sv_scannet200_CA/，naN appears in the loss obtained by training. 
    After debugging, I found that in this command, mask_bce_losses.append(F.binary_cross_entropy_with_logits(
                pred_mask, tgt_mask.float()))，pred_mask and tgt_mask.float() sometimes become tenser[], which leads to loss is naN. 
    Do you konw how to solve this problem？
![2024-09-27 14-17-09 的屏幕截图](https://github.com/user-attachments/assets/c0cc4cbc-1688-48de-83ab-c9eeb7ac6373)
![2024-09-27 14-36-03 的屏幕截图](https://github.com/user-attachments/assets/13a320a1-e5b6-4700-93ab-19df7ec3af30)

**Comment by xuxw98** [comment]
That's ok. This nan will not affect the training.
